content/20.treatments.md

```diff
@@ -110,27 +110,62 @@ Summarize the state of the antiviral approach.
 
 ### Biological Drugs for COVID-19
 
-#### Antibodies
+#### Neutralizing Antibodies
 
 Monoclonal antibodies (mAbs) have revolutionized the way we treat human diseases. 
-As a result, they have become one of the best-selling drugs in the pharmaceutical market in recent years [@doi: 10.1186/s12929-019-0592-z].
+As a result, they have become some of the best-selling drugs in the pharmaceutical market in recent years [@doi: 10.1186/s12929-019-0592-z].
 There are currently 79 FDA approved mAbs on the market including antibodies for viral infections (e.g. Ibalizumab for HIV and Palivizumab for RSV) [@doi:10.1186/s12929-019-0592-z; @doi:10.1146/annurev-immunol-032712-095916].
 Although vaccines remain the most important way to treat viral infections, their development process is long and they fail to provide immediate prophylactic protection or treat ongoing infections [@doi:10.3390/v10120680].
```

Comment:
Hey @nilswellhausen! Great to work with you. This is really good. I will review the section and make some suggestions. The first i this line. Do you meant to say vaccines will treat and prevent? It seems like it should be prevent when referring to prophylaxis. I will let you clarify.

Reply:
I'm here and happy to help! Let me know what I can do :)

Reply:
Yes maybe changing treat to prevent would be better! I don't know how to change things either at this point so its all up to @rando2 haha